tools/__tasks__/compile/fonts/index.config.js

```diff
@@ -1,973 +1,507 @@
+                    'GuardianTextEgyptian/noalts-not-hinted/GuardianTextEgyptian-Bold.ttf',
             },
         ],
     },
+    // *** Guardian Text Egyptian - woff2 - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebAutoHinted.woff2.json',
-        'font-family': '"Guardian Text Egyptian Web"',
+        dest: 'GuardianTextEgyptianHinted.woff2.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 src:
-                    'hinting-auto/kerning-on/original/GuardianTextEgyptianWeb/GuardianTextEgyptianWeb-Regular.woff2',
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff2',
             },
             {
                 'font-style': 'italic',
                 src:
-                    'hinting-auto/kerning-on/latin1/GuardianTextEgyptianWeb/GuardianTextEgyptianWeb-RegularItalic.woff2',
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff2',
             },
             {
                 'font-weight': '700',
+                'font-style': 'italic',
                 src:
-                    'hinting-auto/kerning-on/latin1/GuardianTextEgyptianWeb/GuardianTextEgyptianWeb-Bold.woff2',
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff2',
             },
         ],
     },
+    // *** Guardian Text Egyptian - woff - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebAutoHinted.woff.json',
-        'font-family': '"Guardian Text Egyptian Web"',
+        dest: 'GuardianTextEgyptianHinted.woff.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
                 src:
-                    'hinting-auto/kerning-on/original/GuardianTextEgyptianWeb/GuardianTextEgyptianWeb-Regular.woff',
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Regular.woff',
             },
             {
                 'font-style': 'italic',
                 src:
-                    'hinting-auto/kerning-on/ascii/GuardianTextEgyptianWeb/GuardianTextEgyptianWeb-RegularItalic.woff',
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-RegularItalic.woff',
             },
             {
                 'font-weight': '700',
+                'font-style': 'italic',
                 src:
-                    'hinting-auto/kerning-on/latin1/GuardianTextEgyptianWeb/GuardianTextEgyptianWeb-Bold.woff',
+                    'GuardianTextEgyptian/noalts-autohinted/GuardianTextEgyptian-Bold.woff',
             },
         ],
     },
+    // *** Guardian Text Egyptian - ttf - Hinted ***
     {
-        dest: 'GuardianTextEgyptianWebAutoHinted.ttf.json',
-        'font-family': '"Guardian Text Egyptian Web"',
+        dest: 'GuardianTextEgyptianHinted.ttf.json',
+        'font-family': '"Guardian Text Egyptian"',
         fonts: [
             {
+                'font-weight': '300',
```

Comment:
Regular weight should always be 400, I think. 300 is for Light. Also weight numbers are missing for Regular weights above for all flavours but ttf hinted.